### ISSUE TITLE
Add new AJIVE code

### ISSUE THREAD

**Issue by idc9** [mvlearn/mvlearn#248]
Add the new version of AJIVE from https://github.com/idc9/mvdr/tree/main/mvdr/ajive. The backed of this new version is much cleaner compared to the old version. The API is also enhanced.

**Comment by agramfort** [comment]
why not asking @idc9 to give us a hand to consolidate the community rather
than copying the code?

>

**Comment by rflperry** [comment]
@agramfort I have spoken with @idc9 and he is starting to make PRs with these methods/updates.

**Comment by agramfort** [comment]
awesome !

>

**Comment by idc9** [comment]
Absolutely! I'm going to submit a PR that adds the new AJIVE code as soon as the new MCCA module (#249) is added.
